android/assets/html-zh-rCN/about1d.html

```diff
@@ -2,14 +2,15 @@
 <html>
 <head>
 <meta charset="UTF-8"/>
-<title> 关于一维条码 </title>
+<title>关于条形码</title>
 <link href="../style.css" rel="stylesheet" type="text/css"/>
 </head>
 <body>
-<p> 传统的条形码，如产品包装上印刷，也被称为一维条形码。常用的有几种类型，包括UPC和EAN。大多数看起来像这样： </p>
+<p> 传统的条形码，也被称为一维码。常见的条形码有几种（比如 UPC 和 EAN）。下面是一种常见的条形码： </p>
 <p class="imgcenter"><img src="../images/big-1d.png"/></p>
-<p> 这些一维条码包含一个独特的代码，它通常描述了一种产品，如CD或一本书。你可以看一下这段代码，在互联网上找到价格，评论等。 </p>
-<p> 如果扫描一本书，你也可以为一个词或短语搜索本书的内容，它出现的地方找到的所有网页： </p>
+<p> 条形码内含有一些数字或字符。商品或书本背后通常就会有条形码，并且每种商品的条码都不相同。你可以搜索商品条码来获得相应商品的价格、评论等信息。</p>
+<p> 如果扫描了图书的条码，还可以使用 Google Book Search 在这本书里找某个单词或短语在哪一页。在中国大陆由于特殊国情暂时无法使用此功能。 </p>
 <p class="imgcenter"><img src="../images/search-book-contents.jpg"/></p>
-<p>谷歌翻译，译。</p></body>
+<p> t123yh 翻译 </p>
```

Comment:
OK, thank you. I usually consolidate credits in `AUTHORS` but don't really mind.